README.md

```diff
@@ -97,6 +100,13 @@ You can put a manifest in the same folder with the data files, or you can put it
 - overwrite: if set to “true”, CLI will upload a data file to overwrite the data file with same name that already exists in the Data Hub target storage. If set to “false”, CLI will not upload a data file if a data file with the same name exists in the Data Hub target storage.
 - dryrun: if set to “true”, CLI will not upload any data files to the Data Hub target storage. If set to “false”, CLI will upload data files to the Data Hub target storage.
 
+### File ID generation
+File.tsv template downloaded from Data Model viewer doesn’t contain file Keys/IDs column, because the system will generate it for you. The generated final manifest will be saved in the same place the file manifest is and has a “-final” suffix . For example, file-final.tsv. 
+Uploader CLI Tool will upload this final manifest for you once all data files have been successfully uploaded. So that you don’t have to upload file manifest in the CRDC submission portal.
+
+However, if you need to update the content of the file manifest, please make sure to edit the final manifest and upload it in the CRDC submission portal. 
```

Comment:
recommending the following changes to this section:
"
If you need to update the content of the file manifest, then edit the final manifest and upload it through the CRDC submission portal.
"